CHANGELOG.md

```diff
@@ -6,6 +6,30 @@
   useful summary for people upgrading their application, not a replication
   of the commit log.
 
+## 1.0.0
+
+This release upgrades to Stylelint 15 and now includes:
+
+* Checks for media query syntax, rules and values
+* Checks for number precision inside CSS functions
+* Fixes for duplicate rules with intentionally different units
+
+Read about breaking changes in the Stylelint [**Migrating to v15.0.0**](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-15.md#breaking-changes) guide.
```

Comment:
```suggestion
Read about breaking changes in the Stylelint [**Migrating to v15.0.0**](https://github.com/stylelint/stylelint/blob/main/docs/migration-guide/to-15.md#breaking-changes) guide.

This new version also adds [a rule which checks that properties are used with correct type of values](https://stylelint.io/user-guide/rules/declaration-property-value-no-unknown/#optional-secondary-options). To help us assess its impact on your projects before adding it to this shared configuration, we encourage you to try it out on your projects and [report any extra configuration you needed on this issue](https://github.com/alphagov/stylelint-config-gds/issues/42).
```

Reply:
Thanks @romaricpascal

It deserved its own section ([see changes](https://github.com/alphagov/stylelint-config-gds/compare/9baa94b3260c1f901a21737ec5d491f14a2bacba..b022e42eb460db4d1d6330f8c4436673b1e28f53)) and I've added some more info to the issue:

* https://github.com/alphagov/stylelint-config-gds/issues/42